### PR TITLE
Made CTA buttons closer to mockups

### PR DESCRIPTION
**Pull request by scott-riley**
Super simple PR for making CTA buttons look correct and adding modifiers for filling icons.

### CODE REVIEW

File: components/Btn/Btn.css [hunk 1]
```diff
@@ -166,6 +167,7 @@
   font-size: 0.9em;
   text-transform: uppercase;
   font-weight: 600;
+  padding: .84rem 3.1rem;
```
Comment (rdjpalmer) [inline components/Btn/Btn.css]: Lovely and magic :tada:

**Comment by rdjpalmer** [comment]
:sheep:

File: components/Btn/Btn.css (hunk 1 above)
Comment (iest) [inline components/Btn/Btn.css]: How is this magic?

File: globals/modifiers.css [hunk 2]
```diff
@@ -120,6 +122,16 @@
...
+/* Color icons */
+.flPrimary {  fill: var(--color-primary);}
+.flSecondary {fill: var(--color-secondary);}
+.flError {    fill: var(--color-error);}
+.flAlert {    fill: var(--color-alert);}
+.flText {     fill: var(--color-text);}
+.flTextLight {fill: var(--color-text-light);}
+.flMuted {    fill: var(--color-muted);}
+.flWhite {    fill: var(--color-white);}
+
```
Comment (iest) [inline globals/modifiers.css]: Don't need these — the `fg` modifiers work on any SVGs that have `fill: currentColour` set on them (which is all `Icon`'s)

**Comment by rdjpalmer** [comment]
FYI @scott-riley, the build should pass. If it doesn't, you'll likely need to rebuild without cache:

screen shot 2016-01-26 at 15 04 22

File: components/Btn/Btn.css (hunk 1 above)
Comment (scott-riley) [inline components/Btn/Btn.css]: Spoil sport

**Pull request merged**